galaxy.yml

```diff
@@ -1,4 +1,3 @@
-namespace: community
```

Comment:
hum, I think this is needed.

Reply:
It is? I'll put it back then. There's a command failing that references community.cassandra. Since it wasn't in the url I assumed it was a relic.